src/components/Modals/CreateMeeting/AppointmentList.js

```diff
@@ -214,6 +261,22 @@ class AppointmentList extends React.Component {
                 Reservation Date
               </Typography>
               {date}
+              <Collapse in={this.state.isInvalidDate}>
+              <small className={classes.alertMessage}>
+                {
```

Comment:
same here: code inconsistency, make sure you are using prettier + eslint on your ide